source/_posts/2015-04-28-open-sourcing-your-apps.markdown

```diff
@@ -0,0 +1,63 @@
+
+When we opened the repo, there was a lot of documentation fix PRs. They're low barriers to entry, and fun to make for people on big projects. We still get them pretty regularly.
+
+### Secrets
+
+There isn't large sections of the our app that we are keeping secret. Though we have discussed ways in which we could. There is however a nice solution to having something different for OSS vs your internal team, API compatible CocoaPods. We [do this](/blog/2014/06/20/artsys-first-closed-source-pod/) for our fonts in all apps. We've talked extensively about our [tools for keeping](/blog/2015/01/21/cocoapods-keys-and-CI/) API keys secret, so no need to go over that twice.
+
+Other than that we had already been opening any good abstractions as CocoaPods for anyone to use. All our dependencies were packaged outside of the app, we had no crazy internal SDKs or anything worth hiding within the codebase.
+
+### Selling the idea
+
+It's one thing to think that it's possible, it's another to do it. I'm glad that I am in a position where I can enact change. I felt no resistence in the process. I kept offering potential aventues for someone to stop me too. I emailed the entire team as I started the process 2 weeks before it happened, I talked to anyone who might write issues or contribute from the design team. As I got further along the process and sent another email out that it was going to happen tomorrow. All I got were 👍 and 🎉s in [GIF](https://itunes.apple.com/us/app/gifs/id961850017?l=en&mt=12) form. So I kept moving forwards till that tweet above.
+
+From our dev team's perspective, this is not a brave new world. Our website, [force](https://github.com/artsy/force-public), is open source. Though they operate under different constraints.
+
+From the perspective of Artsy, even though opening our code aligns very strongly with our [values of openness](/blog/2015/03/31/the-culture-of-openness-artsy-mobile/) we are still a company. Opening up our codebase lets our competitors see what we're up to in advance, and [closed source](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html) is the norm for apps. Opening our code and process is also opening ourselves to criticism. 
+
+
+### Evolution of the team
+
+3 months on the way we operate has changed. We're a lot more organized, and the Eigen repo is easily the most well run project on the mobile team. It has [active milestones](https://github.com/artsy/eigen/milestones), that represent long term goals and the current sprint. We discuss a lot of the interesting cultural choices publicly on [issues](https://github.com/artsy/eigen/issues/221) and in our [mobile team repo](https://github.com/artsy/mobile/issues). Having this app in the open, and the experience of doing so has also improved our workflow on other apps. Eidolon for example now runs with a similar structure.
```

Comment:
Add screenshots for some of these things, people won't click, but I think it would be nice to show some examples of active conversations.